roles/webserver/templates/unicorn_service.j2

```diff
@@ -9,7 +9,7 @@ PIDFile={{ app_path }}/shared/tmp/pids/unicorn.pid
 WorkingDirectory={{ current_path }}
 EnvironmentFile={{ environment_file }}
 
-ExecStart=/home/timeoverflow/.rbenv/bin/rbenv exec bundle exec {{ current_path }}/bin/unicorn_rails -c {{ current_path }}/config/unicorn.rb
```

Comment:
we were using a binstub preceded by `bundle exec` which makes the binstub pointless.